airflow/providers/google/cloud/links/dataprep.py

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing import TYPE_CHECKING
+
+from airflow.providers.google.cloud.links.base import BaseGoogleLink
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+BASE_LINK = "https://clouddataprep.com"
```

Comment:
This may not work with my recent change: https://github.com/apache/airflow/pull/26057 (because so far "https://console.cloud.google.com" was the base for all services). Please check if https://console.cloud.google.com can be used instead of https://clouddataprep.com and if not, then update the logic in the ``BaseGoogleLink`` class so the base url is added only when the url doesn't start with http (currently we check if it does start with ``https://console.cloud.google.com`` and desired would be just "http" so it would match different bases).

Reply:
Changed logic in the base class